main.go

```diff
@@ -69,13 +70,23 @@ func main() {
 	}
 
 	for {
+		onlyDelimitCheck := false
 		content, readAllErr := buf.ReadBytes('\n')
 		if readAllErr != nil && readAllErr != io.EOF {
 			fmt.Fprintln(os.Stderr, readAllErr)
 			os.Exit(1)
 		}
 
 		contentLineSplit := bytes.Split(content, []byte{'\n'})
+
+		if *onlyDelimitParameter {
+			for y := range contentLineSplit[0] {
+				if contentLineSplit[0][y] == delimiter[0] {
+					onlyDelimitCheck = true
+				}
+			}
```

Comment:
Isn't there a smarter way to check whether a contains b?

Reply:
bytes.Contains?

Reply:
Exactly.